app/controllers/help_controller.rb

```diff
@@ -1,3 +1,13 @@
 class HelpController < ApplicationController
-  def show; end
+  def show
+    @bank_questions = search(params[:keyword])
+  end
+
+  def search(keyword)
```

Comment:
Let's make this a `private` method since the only place this will be called in is the `show` method.

Reply:
Done